opencog/util/backtrace-symbols.h

```diff
@@ -10,12 +10,14 @@ extern "C" {
 char **oc_backtrace_symbols(void *const *buffer, int size);
 void oc_backtrace_symbols_fd(void *const *buffer, int size, int fd);
 #else
+#ifndef CYGWIN
 #include <execinfo.h>
```

Comment:
pleas don;'t place private nicludes into public header files. Nothing outside of backtrace-symbols.c needs execinfo.h therefore, this should not be placed in this header file. It just pollutes the namespace, and slows compilation.